packages/block-editor/src/components/line-height-control/utils.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { isUndefined } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { __experimentalUseBlockEditProps as useBlockEditProps } from '../../store';
+
+export const BASE_DEFAULT_VALUE = 1.5;
+export const INITIAL_VALUE = '';
```

Comment:
It seems weird to have `INITIAL_VALUE` be a string when the attribute is supposed to be an integer. Perhaps setting it to `null` would make more sense?

Reply:
@ZebulanStanphill That's a great observation! I just pushed some refactors and added comments to clarify it a bit:

```
/**
 * There are varying value types within LineHeightControl:
 *
 * {undefined} Initial value. No changes from the user.
 * {string} Input value. Value consumed/outputted by the input. Empty would be ''.
 * {number} Block attribute type. Input value needs to be converted for attribute setting.
 *
 * Note: If the value is undefined, the input requires it to be an empty string ('')
 * in order to be considered "controlled" by props (rather than internal state).
 */
```